torch/tensor.py

```diff
@@ -807,31 +807,46 @@ def align_to(self, *names):
             [name for name in names if not is_ellipsis(name)],
             ellipsis_idx)
 
-    def unflatten(self, dim, namedshape):
-        r"""Unflattens the named dimension :attr:`dim`, viewing it in the shape
-        specified by :attr:`namedshape`.
+    def unflatten(self, dim, sizes):
+        r"""Expands the dimension :attr:`dim` of the :attr:`self` tensor over multiple dimensions
+        of sizes given by :attr:`sizes`.
+
+        * :attr:`sizes` is the new shape of the unflattened dimension and it can be a `Iterable[int]` as well
```

Comment:
Can it actually be an `Iterable[int]`? iirc our PythonArgParser doesn't handle Iterable[int]. If unflatten can't take in an iterable of ints, I would just say tuple of ints here.

Here's how to create an iterable of ints: `iter([2, 3, 4])`

Reply:
@zou3519  Ah, this is something I don't know......

Reply:
Hmm, so we only support tuple, list and torch.Size is that right?